spec/factories/events.rb

```diff
@@ -2,6 +2,7 @@
   factory :event do
     name { Faker::Hipster.sentence }
     date { Faker::Date.between(from: 10.years.ago, to: Date.today) }
+    week_number { 22 }
```

Comment:
```suggestion
    week_number { Random.rand(0..52)}
```
is there a week 0? Maybe it's 1-52?